src/utils/license.ts

```diff
@@ -54,8 +54,8 @@ export const getFullLicenseName = (
  * @param license - the license slug to check
  * @returns `false` if `license` is 'cc0' or 'pdm', `true` otherwise
  */
-export const isLicense = (license: License): boolean =>
-  !(PUBLIC_DOMAIN_MARKS as ReadonlyArray<License>).includes(license)
+export const isLicense = (license: string): boolean =>
+  !(PUBLIC_DOMAIN_MARKS as ReadonlyArray<string>).includes(license)
```

Comment:
I wonder if the `ReadonlyArray` annotation here is necessary? It seems like noise to me...

```suggestion
  !(PUBLIC_DOMAIN_MARKS as string[]).includes(license)
```

Reply:
Without ReadonlyArray this causes TS error:
`TS2352: Conversion of type 'readonly ["pdm", "cc0"]' to type 'string[]' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.   The type 'readonly ["pdm", "cc0"]' is 'readonly' and cannot be assigned to the mutable type 'string[]'.`